.github/workflows/codeql.yml

```diff
@@ -8,22 +8,19 @@ on:
   schedule:
     - cron: '31 11 * * 2'
 
+permissions: read-all
```

Comment:
Wouldn't this line cover the changes below?

Reply:
You are absolutely correct, I'll update! Thanks :D

Reply:
I've pushed an update with your suggestion 👍 .